src/decisionengine/framework/modules/SourceProxy.py

```diff
@@ -11,12 +10,15 @@
 import decisionengine.framework.dataspace.dataspace as dataspace
 from decisionengine.framework.modules import Source
 from decisionengine.framework.modules.Source import Parameter
-from decisionengine.framework.modules.translate_product_name import translate_all
```

Comment:
I'm not sure why this line is being removed...

It looks like your patch is trying to revert https://github.com/HEPCloud/decisionengine/commit/6991f65f4ed6cea21198bd623180ffe9c9a086f9 for some reason....

Reply:
I don't understand either. I have added it back in.